src/main.js

```diff
@@ -1,39 +1,57 @@
-const { app, BrowserWindow } = require('electron');
+const { app, BrowserWindow, Notification } = require("electron");
 
 // Handle creating/removing shortcuts on Windows when installing/uninstalling.
-if (require('electron-squirrel-startup')) { // eslint-disable-line global-require
+if (require("electron-squirrel-startup")) {
+  // eslint-disable-line global-require
   app.quit();
 }
 
+let mainWindow;
+const url = require("url");
+const path = require("path");
 const createWindow = () => {
   // Create the browser window.
-  const mainWindow = new BrowserWindow({
+  mainWindow = new BrowserWindow({
     width: 800,
     height: 600,
+    webPreferences: {
+      nodeIntegration: true,
+      enableRemoteModule: true
+    },
   });
 
   // and load the index.html of the app.
-  mainWindow.loadURL(MAIN_WINDOW_WEBPACK_ENTRY);
-
+  mainWindow.loadURL(
+    url.format({
+      pathname: path.join(__dirname, "home.html"),
+      protocol: "file:",
```

Comment:
This line, I think, is what is causing a warning upon starting up the application:

![image](https://user-images.githubusercontent.com/4177727/101700883-b1299400-3a32-11eb-952f-ea275db43cf7.png)

In the future (ok if we don't worry about it today) we need to take care of this, for example using some of the solutions discussed at https://github.com/electron/electron/issues/4867#issuecomment-373957680.